exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -91,7 +91,22 @@ drill.exec: {
     use.ip : false
   },
   optimizer: {
-    implementation: "org.apache.drill.exec.opt.IdentityOptimizer"
+    implementation: "org.apache.drill.exec.opt.IdentityOptimizer",
+    enhanced_defaults: {
```

Comment:
Could not work out, in the limited time available, how to pass a system option (which can change at runtime) into the default rules (which appears to not have visibility to the query context.)

Any suggestions for how to pass along the query context so we can get at the runtime options?